actions/pacstrap_action.go

```diff
@@ -0,0 +1,126 @@
+/*
+Pacstrap Action
+
+Construct the target rootfs with pacstrap tool.
+
+Yaml syntax:
+ - action: pacstrap
+   mirror: URL
+
+Optional properties:
+
+- mirror -- URL with ArchLinux-compatible repository
+ If no mirror is specified debos will use http://mirrors.kernel.org/archlinux as default.
+*/
+package actions
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/go-debos/debos"
+)
+
+const pacmanConfig = `
+[options]
+RootDir  = %[1]s
+CacheDir = %[1]s/var/cache/pacman/pkg/
+GPGDir   = %[1]s/etc/pacman.d/gnupg/
+HookDir  = %[1]s/etc/pacman.d/hooks/
+HoldPkg  = pacman glibc
+Architecture = auto
+SigLevel = Required DatabaseOptional TrustAll
+
+[core]
+Server = %[2]s/$repo/os/$arch
+
+[extra]
+Server = %[2]s/$repo/os/$arch
+
+[community]
+Server = %[2]s/$repo/os/$arch
+`
+
```

Comment:
Don't provide the root dir (original comment [here](https://github.com/santoshmahto7/debos/commit/f57f91c3f9936308506f3c4aa66ec92141459b06#r53535960))

Reply:
We do have to provide the GPG dir though, because it needs to be writable on some code paths.

Reply:
Agreed.